configure.ac

```diff
@@ -888,6 +892,22 @@ if test x$GCC = xyes && test $mutt_cv_warnings = yes; then
 	CFLAGS="-Wall -pedantic -Wno-long-long $CFLAGS"
 fi
 
+AC_ARG_ENABLE([fsanitize-ubsan],
+  [AS_HELP_STRING([--enable-fsanitize-ubsan], [Turn on Undefined Behavior Sanitizer (for developers)])],
+  [gl_cc_sanitize_ubsan=yes], [gl_cc_sanitize_ubsan=no])
+
+AC_ARG_ENABLE([fsanitize-asan],
+  [AS_HELP_STRING([--enable-fsanitize-asan], [Turn on Address Sanitizer (for developers)])],
+  [gl_cc_sanitize_asan=yes], [gl_cc_sanitize_asan=no])
+
+if test "$gl_cc_sanitize_ubsan" = yes; then
+  CFLAGS="$CFLAGS -fsanitize=undefined -fno-sanitize-recover=undefined"
+fi
+
+if test "$gl_cc_sanitize_asan" = yes; then
+  CFLAGS="$CFLAGS -fsanitize=address -fsanitize-memory-track-origins"
+fi
+
```

Comment:
As these options are 'off-by-default' and don't have anything in the 'else' clause, can we abbreviate them -- move the CFLAGS into the AC_ARG_ENABLE?
```
AC_ARG_ENABLE([fsanitize-ubsan],
    [AS_HELP_STRING([--enable-fsanitize-ubsan], [Turn on Undefined Behavior Sanitizer (for developers)])],
    [CFLAGS="$CFLAGS -fsanitize=undefined -fno-sanitize-recover=undefined"])
```